remoteappmanager/application.py

```diff
@@ -149,6 +149,16 @@ def start(self):
 
         tornado.ioloop.IOLoop.current().start()
 
+    def urlpath_for_object(self, object):
```

Comment:
Perhaps a subjective comment, I am not sure about the merit of having one function that simply calls another function.